src/main/resources/application.properties

```diff
@@ -1,6 +1,15 @@
 spring.application.name=karuna_backend
 spring.profiles.active=@active.profile@
 
+#prefix for each endpoint
+server.servlet.contextPath=/api
```

Comment:
/api is bad it not should be in path its bad practice
https://opensource.zalando.com/restful-api-guidelines/